components/xsd-fu/templates-cpp/MetadataRetrieve.template

```diff
@@ -424,7 +423,6 @@ namespace ome
         virtual bool
         getPixelsBinDataBigEndian(index_type imageIndex,
                                   index_type binDataIndex) const = 0;
-
```

Comment:
Please don't drop newlines in the templates since it's unnecessary and makes the generated source diff hard to review.